azure-pipelines/main.yml

```diff
@@ -1,23 +1,23 @@
-# Changes in master
+# Changes in main
 # Run on all platforms latest versions
 jobs:
   - template: latest-version.yml
     parameters:
-        name: Linux_master
+        name: Linux_main
         vmImage: 'ubuntu-latest'
-        condition: and(eq(variables['Build.SourceBranch'], 'refs/heads/master'), not(eq(variables['Build.Reason'], 'Schedule')))
+        condition: and(eq(variables['Build.SourceBranch'], 'refs/heads/main'), not(eq(variables['Build.Reason'], 'Schedule')))
         worker: 'standard.yml'
 
   - template: latest-version.yml
     parameters:
-        name: macOS_master
```

Comment:
-R,  --repo <[HOST/]OWNER/REPO>
Select another repository using the [HOST/]OWNER/REPO format